SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -444,6 +444,11 @@ FRepChangeState USpatialActorChannel::CreateInitialRepChangeState(TWeakObjectPtr
 				const bool bRepActorMovement = Cmd.Type == ERepLayoutCmdType::RepMovement && Actor->GetReplicatedMovement().bRepPhysics;
 				if (!Parent.Active && !bRepActorMovement)
 				{
+					if (Cmd.Type == ERepLayoutCmdType::DynamicArray)
+					{
+						CmdIdx = Cmd.EndCmd;
```

Comment:
Could you add a comment explaining what's happening? (i.e. skipping past all the commands in the array because of custom rep condition)